### ISSUE TITLE
`pf-passkey` icon not found

### ISSUE THREAD

**Issue by austincarpenter** [rawilk/profile-filament-plugin#21]
### profile-filament-plugin Version

v0.3.2

### Laravel Version

v10.46.0

### Bug description

`Svg by name "pf-passkey" from set "default" not found.`

The custom Blade icons set that the package registers does not exist when it comes to resolving the icon. For now my workaround is to register the default set per the blade icons config and place a duplicate of `pf-passkey.svg` in my own `/resources/svg` folder.

### Steps to reproduce

Install package as per instructions.

**Comment by rawilk** [comment]
I haven't run into this issue myself, but I'll see if I can replicate this when I get a chance to. The icons should be getting registered from the package's service provider. Currently it's done like this:

```php
$sets = $this->app['config']->get('blade-icons.sets');
$sets['profile-filament'] = [
    'path' => 'vendor/rawilk/profile-filament-plugin/resources/svg',
    'prefix' => 'pf',
];

$this->app['config']->set('blade-icons.sets', $sets);
```

If there's a better way of registering them, I'd be open to changing that.

**Comment by rawilk** [comment]
This issue should be resolved with `v0.5.0`, which I should be releasing soon. I changed how the package's icons are registered, which should prevent this error.
